src/test/java/problems/impl/LargeValuesAdderImplTest.java

```diff
@@ -0,0 +1,35 @@
+package problems.impl;
+
+import org.junit.Assert;
+import org.junit.Test;
+import problems.interfaces.LargeValuesAdder;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class LargeValuesAdderImplTest {
+
+    private final LargeValuesAdder largeValuesAdder = new LargeValuesAdderImpl();
+    private final List<Character> characterList1 = new ArrayList<>(Arrays.asList('9', '9'));
+    private final List<Character> characterList2 = new ArrayList<>(Arrays.asList('9', '9', '9'));
+    private final List<Character> characterList3 = new ArrayList<>(Arrays.asList('1'));
+
+    @Test
+    public void testExpected() {
```

Comment:
Test names should reflect what you're testing... either use `shouldAdd`, `itShouldAdd` or `testShouldAdd`, depending on what name format you prefer.